pkg/admit/appconfig.go

```diff
@@ -158,9 +158,23 @@ func (a *Admit) checkComponent(appConf *v1alpha1.ApplicationConfiguration) error
 				}
 			}
 			if !found {
-				return fmt.Errorf("component %s don't have this parameter %s", v.ComponentName, p.Name)
+				return fmt.Errorf("component %s don't have this parameter %s", component.ComponentName, p.Name)
+			}
+		}
+		for _, t := range component.Traits {
+			if t.Name == "volume-mounter" {
+				for _, c := range comp.Spec.Containers {
+					for _, v := range c.Resources.Volumes {
+						//TODO: check the volumeName from the Trait to match the disk volume name
+						if v.Disk.Ephemeral{
```

Comment:
gofmt?

Reply:
how to link to the issue other than paste the link here?

Reply:
Just paste it here is good enough.
Also the code looks not "gofmt-ed"?

Reply:
thanks, go fmted.